translator/translate/otel/processor/metricstransformprocessor/translator.go

```diff
@@ -11,9 +11,27 @@ import (
 	"go.opentelemetry.io/collector/confmap"
 	"go.opentelemetry.io/collector/processor"
 
+	"github.com/aws/amazon-cloudwatch-agent/internal/containerinsightscommon"
 	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
 )
 
+const gpuLogSuffix = "GPU"
+
+var metricDuplicateTypes = []string{
+	containerinsightscommon.TypeContainer,
+	containerinsightscommon.TypePod,
+	containerinsightscommon.TypeNode,
+}
```

Comment:
Instead of (confusingly) reusing the old metric types, please update `internal/containerinsightscommon/MetricName()` to understand your new GPU metric types.  Then for `"new_value"` below you can use the new types as well, instead of `t + gpuLogSuffix`.